src/markup/codemirror/create.ts

```diff
@@ -152,7 +153,38 @@ export function createCodemirror(params: CreateCodemirrorParams) {
                 onScroll(event);
             },
             paste(event, editor) {
-                if (event.clipboardData && parseInsertedUrlAsImage) {
+                if (!event.clipboardData) return;
+
+                // Note: I have editor.dispatch() in the try-catch on purpose.
+                // The code's pretty new and there might be random issues we haven't caught yet,
+                // especially with invalid HTML or weird DOM parsing errors.
+                // If something goes wrong, I just want to fall back to the "default pasting"
+                // rather than break the entire experience for the user.
+                // It’s kind of like a temporary safety net right now until things are more stable.
+                try {
+                    // Handle HTML insertion
+                    const htmlContent = event.clipboardData.getData(DataTransferType.Html);
```

Comment:
I think you should check the yfmContent by using `event.clipboardData.getData(DataTransferType.Yfm)` and use it before other checks

Reply:
Done it, but here is one interesting thing.

If i copy WYSIWYG content and paste in in markdown mode, i will lose all formatting because what will be in copy buffer is text/html and not text/yfm

https://github.com/user-attachments/assets/014a0b4a-a78d-482e-9b0a-b740317eb19b


Actually i was not able to get  text/yfm at any point in time .. mb there is some bug?

Reply:
fixed